### ISSUE TITLE
The `errorProps` aren't working in any component that depends on `Input.Wrapper`

### ISSUE THREAD

**Issue by A-Marzoug** [mantinedev/mantine#2603]
### What package has an issue

@mantine/core

### Describe the bug

The bug is described as follows: If I set `size="xl"` in the `TextInput` component and give the object `{size : "xs"}` as `errorProps` for the same component, the error size stays unchanged.

### What version of @mantine/hooks page do you have in package.json?

5.5.0

### If possible, please include a link to a codesandbox with the reproduced problem

https://codesandbox.io/s/black-wood-y55brg

### Do you know how to fix the issue

Yes

### Are you willing to participate in fixing this issue and create a pull request with the fix

Yes

### Possible fix

Change line 143 in `InputWrapper.tsx`

```
// from this
<InputError {...errorProps} key="error" {...sharedProps}>

// to this
<InputError key="error" {...sharedProps} {...errorProps}>
```
Because the way the props are ordered is wrong the shared props will always overwrite the error props

**Comment by rtivital** [comment]
Fixed in 5.5.1

**Comment by kenzaflow** [comment]
I have exactly this problem, I think.

Here's where it's wrong:

https://github.com/mantinedev/mantine/blob/3a4f65d9c8b9a217c5dd4c3e078272181b2c8e47/packages/%40mantine/core/src/components/Input/InputWrapper/InputWrapper.tsx#L217
